build-plugin/README.md

```diff
@@ -88,11 +89,34 @@ jobs:
           attestation: true # new line
 ```
 
+## Changelog Generation
+
+If you pass `use_changelog_generator: true` to the action, it will:
+
+1. Generate a changelog using the github-changelog-generator tool
```

Comment:
Nit/Suggestion: Same here, I would add a link to the github-changelog-generator tool